Package.swift

```diff
@@ -29,7 +29,7 @@ let package = Package(
             )
         ],
     dependencies: [
-        .package(url: "https://github.com/IBM-Swift/Kitura-Credentials.git", .upToNextMinor(from: "2.0.0")),
+        .package(url: "https://github.com/IBM-Swift/Kitura-Credentials.git", from: "2.0.0"),
```

Comment:
Let's make this `.upToNextMinor(from: "2.1.0")` so that we can tag Kitura-Credentials as `2.1.0` instead of being forced to tag it `3.0.0` (since it removes SwiftyJSON which this package is relying on **indirectly**). Keeping the version range to minors will mean we have a bit more maintenance to do keeping the `Package.swift` up to date, but hopefully it will reduce the likelihood of slow/hang resolution problems with SPM.